File: CursoPOOUber/Java/curso/Car.java
class Car{
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
         this.license = license;
         this.driver = driver;
    }

    void printDataCar(){
        System.out.println("License: "+ license + " Name Driver: "+ driver.name + " Passenger: "+passenger);
     }

     public Integer  getPassenger(){
         return passenger;
     }

     public void setPassenger(Integer passenger){
         if(passenger ==4)
            this.passenger = passenger;    
        else
            System.out.println("Necesitas asignar 4 pasajeros ");
     }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
}

File: CursoPOOUber/PHP/Car.php
<?php
require_once 'Account.php'
class Car{
  public $id;
  public $license;
  Account $driver;
  public $passenger;

  function __construct($license , Account $driver){
    $this->license  = $license;
    $this->driver = $driver;
  }

  public function printDataCr(){
     echo "License: ".$this->license . "Driver: ".$this->driver->name;
  }
}